### PR TITLE
Adds immersion and realism to internals tank suicides

### PR DESCRIPTION
**Pull request by SteelSlayer**
## About The Pull Request
When you suicide with an internals tank, normally you just fall over and gib immediately. Now you get to see people inflate from inhaling the gas in the tank before they explode. I can still tweak how it looks but this is what I have so far:

GIF

## Changelog
:cl:
tweak: When you suicide using an internals tank, you're character will now inflate, and grow in size before you gib.
code: Adds a safe_gib argument to the carbon gib proc. Allows you to specifiy if you want to preserve the items on the mob you're gibbing.
/:cl:

### CODE REVIEW

**Comment by Tumesuo** [comment]
Fetish content, speedmerge.

**Comment by nemvar** [comment]
This doesn't make people uncloneable after they commit die.

**Comment by SteelSlayer** [comment]
@nemvar Is it the MANUAL_SUICIDE_NONLETHAL that breaks it?

**Comment by nemvar** [comment]
Yep, that return value requires you to manually set the suicide flag.

**Comment by TheVekter** [comment]
Definitely fetish material. I'm ~~aroused~~ disgusted.

**Comment by PepperPrepper** [comment]
I'd add this to mint toxins as well

**Comment by wesoda25** [comment]
Does mint toxin even work

**Comment by Shadowflame909** [comment]
>Does mint toxin even work 

Yes.

Surprising to say, I don't hate this. Even though its fetish content, all the good times of watching tom and jerry cartoons overwrite the negative connotation. 

Nostalgia > Furrys

**Comment by SteelSlayer** [comment]
Everyone saying this is fetish content is probably fucked in the head, and I suggest you take a break from the internet. Also what do furries have to do with this? :thinking:

**Comment by Shadowflame909** [comment]
I don't want to assume by your sprite..

And how you code on the paradise code-base.

**Comment by PepperPrepper** [comment]
I agree with @SteelSlayer How the fuck is this fetish content?

I mean I get the whole everything that is weird is fetish content joke we have running, But if anyone think this is some form of fetish is just being stupid.

Who takes sexual pleasure in watching someone/being inflated till they pop?

**Comment by Couls** [comment]
> I don't want to assume by your sprite..
> 
> And how you code on the paradise code-base.

How is this related to the PR though? someone's profile pic or past works shouldn't affect the content of the PR it doesn't become fetish content because someone has a lewd profile pic but posts a PR

**Comment by Shadowflame909** [comment]
> 
> 
> I agree with @SteelSlayer How the fuck is this fetish content?
> 
> I mean I get the whole everything that is weird is fetish content joke we have running, But if anyone think this is some form of fetish is just being stupid.
> 
> Who takes sexual pleasure in watching someone/being inflated till they pop?

There's literally an ss13 server called Vorestation.

/TG/ has merged PRs getting rid of anything related to that. Like fat, humans being able to eat monkeys, and Xenos doing the same.

Is it innocence?!? Or is it argumentative ignorance to ignore the horrifying truth of the matter!

**Comment by wesoda25** [comment]
Sorry shadowflame i dont quite follow could you please elaborate?

**Comment by TheVekter** [comment]
Holy shit I was joking guys, are you serious?

**Comment by Shadowflame909** [comment]
I'm not going to tell the tale of one of 4chans greatest enemies on github because that's dumb and would stray away from the PR.

I'd be happy to do it on the forums though.

All in All, to save you all the time and strain of losing your innocence.

An Inflation fetish is a very very very real thing.

**Comment by SteelSlayer** [comment]
You'll have to forgive me as I'm new to the codebase and I don't know all the memes yet. 

But anyway, this is not intended as fetish content. I have no desire to put any ERP, or vore content into the game. This is just a 3 second animation I made for fun testing out some new stuff I've never worked with before, that's it.

**Comment by TheVekter** [comment]
I legit do _not_ care if someone comes on our server, kills themselves with an airtank, and gets a semi from it. You can only do it once a round, it's nowhere near as concerning as ERP or those goddamn locking outfits, it's a cool animation that has more than one use. If people haven't been using the mint to jerk off, they're not gonna use this.

Merge it, it looks great, it's a funny suicide that could use more work making it look better. Hard +1.

**Comment by SteelSlayer** [comment]
Yeah I'll see about doing that, and maybe adding this to the mint toxin too.

**Comment by anconfuzedrock** [comment]
I accidentally made it easy to do airtank suicides again, I need to make the pressure requirement higher now.

**Comment by Tumesuo** [comment]
God how will i able to play as hop without popping a boner? Can you make it so that wizards can timestop to make the animation freeze? :flushed:

**Comment by SteelSlayer** [comment]
Alright I made some final adjustments, and added the animation to the mint gib for fat people too. I'm not gonna add the floor tile damage as none of the other gibs do that.. plus I don't know how to without causing an explosion :smile:. 

I'm going to call this completed and ready for review/merge.

**Comment by PKPenguin321** [comment]
the PR that made dobson join /tg/station

**Comment by Shadowflame909** [comment]
> the PR that made dobson join /tg/station

Told you zoomers it's 4chan history

**Comment by Timonkeyn** [comment]
I still don't understand how you consider xeno eating humans fetish content, as it was a useful tool and xeno queen dragging someone is basically walk crawl speed right now

**Comment by Couls** [comment]
speedmerge so people shut up about fetish content

File: code/modules/mob/living/carbon/death.dm [hunk 1]
```diff
@@ -17,12 +17,23 @@
...
 
-/mob/living/carbon/gib(no_brain, no_organs, no_bodyparts)
+/mob/living/carbon/proc/inflate_gib() // Plays an animation that makes mobs appear to inflate before finally gibbing
+	addtimer(CALLBACK(src, .proc/gib, null, null, TRUE, TRUE), 25)
+	var/matrix/M = matrix()
+	M.Scale(1.8, 1.2)
+	animate(src, time = 40, transform = M, easing = SINE_EASING)
+
+/mob/living/carbon/gib(no_brain, no_organs, no_bodyparts, safe_gib = FALSE)
+	if(safe_gib) // If you wan't to keep all the mob's items and not have them deleted
```
Comment (actioninja) [inline code/modules/mob/living/carbon/death.dm]: tiny, but `wan't`

**Review by actioninja — CHANGES_REQUESTED** [review]

**Pull request merged**